backend/321SEVER_SIMON.js

```diff
@@ -0,0 +1,569 @@
+var express = require('express'); 
+var app = express(); 
+  
+const bodyParser = require('body-parser'); 
+const { application } = require('express');
+
+const {MongoClient} = require("mongodb");
+const { response } = require("express");
+const e = require('express');
+const uri ="mongodb://localhost:27017"
+const client = new MongoClient(uri)
+
+app.use(bodyParser.json()); 
+app.use(bodyParser.urlencoded({ extended: true })); 
+  
+
+//create account with google
+app.post('/google_sign_up', async(req, res) => { 
+   //var user_id = generate_new_user_id(); // write a method to generate a user id for new user
+    const user_gmail = req.body.email;
+    const userName= req.body.userName;
+    const result= await client.db("user").collection("users").find({_id : user_gmail}).toArray();
+    console.log(result);
+    if(result.length === 0){
+        await client.db("user").collection("users").insertOne({

```

Comment:
Codacy has a fix for the issue: [Replace `········await·client.db("user").collection("users").insertOne({␍` with `····await·client.db("user").collection("users").insertOne({`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion
    await client.db("user").collection("users").insertOne({
```